apps/src/redux/instructions.js

```diff
@@ -163,10 +163,11 @@ export const setHasAuthoredHints = hasAuthoredHints => ({
  * @param {string} locale
  * @param {boolean} noInstructionsWhenCollapsed
  * @param {boolean} showInstructionsInTopPane
+ * @param {boolean} hasInputOutputTable
  * @returns {Object}
  */
 export const determineInstructionsConstants = (instructions, markdownInstructions,
-    locale, noInstructionsWhenCollapsed, showInstructionsInTopPane) => {
+    locale, noInstructionsWhenCollapsed, showInstructionsInTopPane, hasInputOutputTable) => {
```

Comment:
Not required, but recommended for near future - now that this function signature is up to six arguments, and the last two are boolean flags, an options object might read better overall - esp. now that we can do elegant destructuring at the top of the function.